integrationtests/server-integration/third-party-server/src/test/java/org/infinispan/test/integration/thirdparty/remote/MemcachedClientIT.java

```diff
@@ -0,0 +1,31 @@
+package org.infinispan.test.integration.thirdparty.remote;
+
+import static org.infinispan.test.integration.thirdparty.DeploymentHelper.addLibrary;
+
+import org.infinispan.test.integration.thirdparty.DeploymentHelper;
+import org.infinispan.test.integration.remote.AbstractMemcacheClientIT;
+import org.infinispan.test.integration.memcached.SimpleMemcachedClient;
+import org.infinispan.test.integration.util.ITestUtils;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+public class MemcachedClientIT extends AbstractMemcacheClientIT {
```

Comment:
Do we need the equivalent of this test for wildfly-modules?

Reply:
We can add it together with the CDI Tests.

Reply:
Ok np. Can you create a Jira for this, or add it the CDI test issue.

Reply:
https://issues.redhat.com/browse/ISPN-12623
https://issues.redhat.com/browse/ISPN-12624